consensus/src/quorum_store/batch_store.rs

```diff
@@ -300,7 +301,7 @@ impl BatchStore {
     }
 
     fn persist_inner(&self, persist_request: PersistedValue) -> Option<SignedBatchInfo> {
-        match self.save(persist_request.clone()) {
```

Comment:
@sitalkedia I made the above change in `insert_to_cache` function to eliminate this clone() operation. Earlier, we were cloning the persist request all the time and then potentially replace the payload with None. Now, I'm cloning the persist request only when it is required.